packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page/entity-type-tabs/tab-link.tsx

```diff
@@ -7,7 +7,7 @@ import {
   typographyClasses,
 } from "@mui/material";
 import { FunctionComponent, ReactElement } from "react";
-import { Link } from "../../../../shared/ui/link";
+import { Link } from "../../../../../../shared/ui/link";
```

Comment:
Side thought for future consideration: it'd be nice to switch to one of these import paths:

```suggestion
import { Link } from "../../../../../../shared/ui";
```

```suggestion
import { Link } from "../../../../../../shared/ui-link";
```

```suggestion
import { Link } from "../../../../../../shared/@ui/link";
```

I know this is out of scope, just leaving a comment as a reminder to the team.